examples/webgpu_refraction.html

```diff
@@ -126,7 +126,7 @@
 
 				// renderer
 
-				renderer = new THREE.WebGPURenderer( { antialias: true } );
+				renderer = new THREE.WebGPURenderer( /*{ antialias: true }*/ );
```

Comment:
@sunag I have to disable also antialias for the example to work as well. Can you provide me context on this?

Reply:
What I did is set the antiAlias to false like here and handle the pixel density manually in the ctx:
```tsx

    const context = ref.current!.getContext("webgpu")!;
    const canvas = context.canvas;
    canvas.width = canvas.clientWidth * PixelRatio.get();
    canvas.height = canvas.clientHeight * PixelRatio.get();
```

let me know if you have any thoughts on this.